src/labels/CustomLabels.labels

```diff
@@ -13800,15 +13808,15 @@ Note: The NPSP Settings tab is visible in the Nonprofit Success Pack application
     </labels>
     <labels>
         <fullName>gsResourcesItemProgramManagementLinkLabel</fullName>
-        <categories>Get Started Page</categories>
+        <categories>tart Managing Programs</categories>
         <language>en_US</language>
         <protected>true</protected>
         <shortDescription>Resource Program Management link label in Get Started Resources section</shortDescription>
         <value>Get Started</value>
     </labels>
     <labels>
         <fullName>gsResourcesItemMarketingEngagementLabel</fullName>
-        <categories>Get Started Page</categories>
+        <categories>Start Engaging Constituents</categories>
```

Comment:
Why did the category change here and below? That should be the same for all GetStarted labels